File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 6 and 7 July 2019/1.Repaining/1.Repaining.js
function repainting(p) {
  let nailon = +p[0]
  let pain = +p[1]
  let razred = +p[2]
  let hours = +p[3]

  let klNailon = (nailon + 2) * 1.50 //18
  let klPain = ((pain * 110 / 100)) * 14.50 //175.45
  let klRazred = razred * 5.00
  let subtotal = klNailon + klPain + klRazred + 0.40 //213.85
  let total = ((subtotal * 0.30) * hours + subtotal).toFixed(2)

  console.log(`Total expenses: ${total} lv.`);
}
//repainting(['10','11','4','8',])

File: Python Exams/01. Basic Exam Collection/Basics Online Exam - 6 and 7 July 2019/1.Repaining/1.Repaining.py
nylon, pain, thinner, hours =  [int(input()) for _ in range(4)]

kg_nylon = (nylon + 2) * 1.50
klPain = (pain * 110 / 100) * 14.50
kg_thinner = thinner * 5.00
subtotal = kg_nylon + klPain + kg_thinner + 0.40
total = subtotal * 0.30 * hours + subtotal

print(f'Total expenses: {total:.2f} lv.')


'''
10
11
4
8
'''
